src/main/resources/hudson/plugins/build_timeout/BuildStepWithTimeout/config_ja.properties

```diff
@@ -1,6 +1,6 @@
 # The MIT License
 #
-# Copyright (c) 2004-2012, Sun Microsystems, Inc., Seiji Sogabe
+# Copyright (c) 2016-, Seiji Sogabe
```

Comment:
Somehow this file is displayed as renamed, they are actually not renamed but removed and added.
LikelyStuckTimeOutStrategy/config.jelly have no texts to translate (there are no text "Likely stuck"), and LikelyStuckTimeOutStrategy/config_ja.properties is unnecessary.